app/workers/articles/resave_for_association_worker.rb

```diff
@@ -0,0 +1,23 @@
+module Articles
+  class ResaveForAssociationWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority, retry: 10, lock: :until_executing
+
+    # @param klass_name [#constantize, String] the name of the class
+    # @param id [Object, Integer] the ID of the record for the given class
+    def perform(klass_name, id)
+      object = klass_name.constantize.find_by(id: id)
```

Comment:
this _could_ raise no method error on invalid job args (or if a class were renamed while jobs were enqueued). Not an objection, just thinking aloud as I read.

Reply:
I'm okay with this failing with a noisy exception